CMakeLists.txt

```diff
@@ -62,7 +62,7 @@
 #######################################################################################################################
 
 # Ubuntu 20.04 ships with CMake version 3.16.3. But CI on Mac is outdated and picking up 3.15.
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
It'd be great if you please update the comment with the mentioned version.

One thing, cmake 3.8 is way too high. Is there any release of 3.8, at all? Please confirm.
cmake 3.16 should work fine, is there any issue?